cs/test/core/Core.csproj

```diff
@@ -56,8 +56,8 @@
     <Compile Include="Util.cs" />
     <Compile Include="XmlTests.cs" />
     <BondCodegen Include="Aliases.bond">
-      <Options Condition="'$(TargetFrameworkVersion)' == 'v4.5'">$(BondOptions) --using="Lazy=Lazy&lt;{0}&gt;" --using="OrderedSet=SortedSet&lt;{0}&gt;" --using="Decimal=decimal" --using="EnumString=Alias.EnumString&lt;{0}&gt;" --using="Array={0}[]" --using=ArrayBlob=byte[] --using="CustomList=UnitTest.Aliases.SomeCustomList&lt;{0}&gt;"</Options>
-      <Options Condition="'$(TargetFrameworkVersion)' == 'v4.0'">$(BondOptions) --using="Lazy=Lazy&lt;{0}&gt;" --using="OrderedSet=SortedSet&lt;{0}&gt;" --using="Decimal=decimal" --using="EnumString=Alias.EnumString&lt;{0}&gt;" --using="Array={0}[]" --using=ArrayBlob=byte[] --using="CustomList=UnitTest.Aliases.SomeCustomList&lt;{0}&gt;"</Options>
+      <Options Condition="'$(TargetFrameworkVersion)' == 'v4.5'">$(BondOptions) --using="Lazy=Lazy&lt;{0}&gt;" --using="OrderedSet=SortedSet&lt;{0}&gt;" --using="Decimal=decimal" --using="EnumString=Alias.EnumString&lt;{0}&gt;" --using="Array={0}[]" --using=ArrayBlob=byte[] --using="CustomList=UnitTest.Aliases.SomeCustomList&lt;{0}&gt;" --using="ConvertedBool=Box&lt;bool&gt;" --using="ConvertedInt8=Box&lt;System.SByte&gt;" --using="ConvertedInt16=Box&lt;System.Int16&gt;" --using="ConvertedInt32=Box&lt;System.Int32&gt;" --using="ConvertedInt64=Box&lt;System.Int64&gt;" --using="ConvertedUInt8=Box&lt;System.Byte&gt;" --using="ConvertedUInt16=Box&lt;System.UInt16&gt;" --using="ConvertedUInt32=Box&lt;System.UInt32&gt;" --using="ConvertedUInt64=Box&lt;System.UInt64&gt;" --using="ConvertedFloat=Box&lt;float&gt;" --using="ConvertedDouble=Box&lt;double&gt;" --using="ConvertedString=Box&lt;string&gt;" --using="ConvertedWString=Box&lt;string&gt;"</Options>
```

Comment:
There's really no reason to do this testing via codegen. If we just use aliases on a class in the unit test .cs file, we don't need changes in the .bond file or the .csproj file.